dex/testing/dcrdex/harness.sh

```diff
@@ -22,12 +22,12 @@ sudo -u postgres -H psql -c "DROP DATABASE IF EXISTS ${TEST_DB}" \
 -c "CREATE DATABASE ${TEST_DB} OWNER dcrdex"
 
 EPOCH_DURATION=15000
-if [ "$#" -eq 1 ]; then
-    if [ "$1" -lt 1000 ]; then
+if [ -n "${EPOCH}" ]; then
```

Comment:
With the new env variable approach, I think it would be ok to use shell parameter expansion instead of a test.

`EPOCH_DURATION=${EPOCH:-1500}` or a variation with just one variable using `${EPOCH_DURATION:=val}`

Reply:
oh, nice,
`EPOCH_DURATION` seem too long to type in terminal, will go with the former